utils/ccloud_library.sh

```diff
@@ -103,7 +105,7 @@ function ccloud::validate_version_ccloud_cli() {
 
   ccloud::validate_ccloud_cli_installed || exit 1
 
-  REQUIRED_CCLOUD_VER=${1:-"1.7.0"}
+  REQUIRED_CCLOUD_VER=${1:-$CCLOUD_MIN_VERSION}
```

Comment:
@rspurgeon `REQUIRED_CCLOUD_VER` is treated like a minimum (see 2 lines below), so can this param be removed, and then use `CCLOUD_MIN_VERSION` instead below?

Reply:
Yes, I can do that. My hesitation is changing global variable assigments in a library because I don't exactly know how it is used by any client (outside this repo or maybe even ones we don't control that someone has built). Our wide use of global variables reduces my confidence in changing code without side effects, so I try to change as little as possible.

Reply:
1. `REQUIRED_CCLOUD_VER` is not exported.
2. "There are no guarantees for backwards compatibility"

tl;dr Risk is low to break things unwittingly.  Let's optimize for reducing maintenance burden on ourselves.

Reply:
All variables in bash are global by default, export is not required for side effects.

Reply:
Yeah, you're right.  I was thinking of the bash script use case, not function use case.

Regardless, no guarantees for backwards compatibility.  Let's please remove `REQUIRED_CCLOUD_VER`

Reply:
it's removed.  I'd like to see us transition to `local` so it's completely safe for function local changes.

Reply:
@rspurgeon would `local` also address the issues encountered when running it back to back?

Reply:
I'm not 100% certain.  Some of our instructions rely on the user's shell having particular values set in order to perform demos ( example ccloud::destroy_ccloud_stack $SERVICE_ACCOUNT_ID).  I'm not sure where everything intersects with the issue of multiple runs and our usage of global vars.  I could certainly research and get back

Reply:
I was asking just out of curiosity ;) to keep in my mind if/when we revisit those issues.  No need to research this now, but thank you for the idea in the first place!